index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <h1 class="title">
+        Hello Microverse!
+    </h1>
+</body>
+</html>
```

Comment:
As [required](https://github.com/microverseinc/curriculum-transversal-skills/blob/9bf76d5f906c3db5f38b26577e52a260ef4191a5/documentation/hello_microverse_project.md?plain=1#L28) for this and most other Microverse activities, kindly set up the linters. For the current project, the [linters for HTML & CSS](https://github.com/microverseinc/linters-config/blob/master/html-css) projects is required.